target/linux/octeon/base-files/etc/board.d/01_network

```diff
@@ -8,6 +8,9 @@
 board_config_update
 
 case "$(board_name)" in
+itus*)
```

Comment:
Please don't use wildcards:
https://openwrt.org/docs/guide-developer/device-support-policies#no_wildcards

Reply:
I need to use itus* or define itusrouter itusgateway itusbridge, and they all have the same default port configuration.

Which should I use?

Reply:
Read the reference. Just replace this by
```
itusbridge|\
itusgateway|\
itusrouter)
```

as already suggested elsewhere (or the more modern names after we have agreed on them).

Reply:
Will correct